nautobot/extras/models/models.py

```diff
@@ -686,6 +689,11 @@ class JobLogEntry(BaseModel):
     def __str__(self):
         return self.message
 
+    class Meta:
+        ordering = ["created"]
+        get_latest_by = "created"
```

Comment:
Thanks. Tagging #1193, though I'd like to keep that open for the moment to also address the formatting of the table column in a future commit.